rotkehlchen/tests/api/test_pnl_csv.py

```diff
@@ -90,94 +92,104 @@ def assert_csv_export_response(response, csv_dir, is_download=False, unicode_che
 @pytest.mark.parametrize('db_settings', [
     {'pnl_csv_with_formulas': False},
 ])
+@pytest.mark.parametrize('encoding_to_use', ['utf-8', 'cp1252'])
 def test_history_export_download_csv(
         rotkehlchen_api_server_with_exchanges,
         tmpdir_factory,
+        encoding_to_use,
 ):
```

Comment:
This is a heavey tests due to the number of events processed and you only test the history export and (afaik) not the other places. Could you create a test only adding the event and trying to export it? In my pc this test takes around ~8 secs and you repeat it twice. I believe executing only the export that you need would be much faster. Also maybe you could also test the other locations where you export data if possible